README.md

```diff
@@ -45,70 +45,73 @@ helm repo update
 
 1. Clone the latest version of kubeslice-controller from  the `master` branch.
 
-```console
-git clone https://github.com/kubeslice/kubeslice-controller.git
-cd kubeslice-controller
-```
+   ```console
+   git clone https://github.com/kubeslice/kubeslice-controller.git
+   cd kubeslice-controller
+   ```
 
 2. Adjust image name variable `IMG` in the [`Makefile`](Makefile) to change the docker tag to be built.
```

Comment:
Modify  the image name variable... 

or you can start replace Adjust with Edit...

Reply:
done